src/probnum/randprocs/markov/discrete/_discrete_gaussian.py

```diff
@@ -8,12 +8,11 @@
 import scipy.linalg
 
 from probnum import config, linops, randvars
+from probnum.randprocs.markov import _transition
+from probnum.randprocs.markov.discrete import _utils
```

Comment:
See comment above. Here, `_utils` is `discrete._utils`

Reply:
let's discuss above :) I will resolve here.